### PR TITLE
Update Joomla to 3.9.21

### PR DESCRIPTION
**Pull request by J0WI**
Changes:

- https://github.com/joomla-docker/docker-joomla/commit/9deecb9: Merge pull request https://github.com/joomla-docker/docker-joomla/pull/102 from J0WI/update-repo-url
- https://github.com/joomla-docker/docker-joomla/commit/0c6aa57: Update repo url
- https://github.com/joomla-docker/docker-joomla/commit/9bbb72b: Joomla 3.9.21 (https://github.com/joomla-docker/docker-joomla/pull/98)
- https://github.com/joomla-docker/docker-joomla/commit/101afd8: update maintainer to @HLeithner (https://github.com/joomla-docker/docker-joomla/pull/99)
- https://github.com/joomla-docker/docker-joomla/commit/97ec165: Update to 3.9.20
- https://github.com/joomla-docker/docker-joomla/commit/b887639: Added optional environment variable JOOMLA_DB_PASSWORD_FILE (https://github.com/joomla-docker/docker-joomla/pull/100)

### CODE REVIEW

File: library/joomla [hunk 1]
```diff
@@ -1,49 +1,49 @@
-# this file is generated via https://github.com/joomla/docker-joomla/blob/65b178365a9294612ba364adb2e12c02df056430/generate-stackbrew-library.sh
+# this file is generated via https://github.com/joomla-docker/docker-joomla/blob/0c6aa57f33a4ac2f371c35467191424730b23078/generate-stackbrew-library.sh
 
 Maintainers: Michael Babker <michael.babker@joomla.org> (@mbabker)
```
Comment (yosifkit) [inline library/joomla:3]: You may want to update your `generate-stackbrew-library.sh` with the new maintainer too.

Comment (J0WI) [reply library/joomla:3]: cc @HLeithner @zero-24

**Comment by zero-24** [comment]
@J0WI isnt that here the PR that should be done automaticly by that bot? Or do we have to do such PR manually too?

**Comment by J0WI** [comment]
> @J0WI isnt that here the PR that should be done automaticly by that bot? 

Yes, normally it's created by @docker-library-bot

File: library/joomla (hunk 1 above)
Comment (zero-24) [reply library/joomla:3]: Sorry did not noticed earlier here is the PR: https://github.com/joomla-docker/docker-joomla/pull/103

**Comment by zero-24** [comment]
> Yes, normally it's created by @docker-library-bot

hmm lets see what happens when the PR for the maintainer update is merged.

**Comment by tianon** [comment]
@docker-library-bot only makes commits, not PRs

**Comment by zero-24** [comment]
Ok so we have to-do an PR on any update to this repo here? Or what is the intended process?

**Comment by tianon** [comment]
The normal process is that you should make changes in your repository, then make a PR here to update the `library/joomla` file to point to your updated repository contents.

In the case of `library/joomla`, we have extended our own @docker-library-bot updating behavior to your repository, so it will make normal bump commits and create a `joomla` branch on its fork of this repository (https://github.com/docker-library-bot/official-images), but it is up to you to create the actual PR, since if we have PR review comments, they will be directed at you as the image maintainer, not at the bot.

When the bot detects changes, what it calculates the updated `library/joomla` to be will be visible at https://github.com/docker-library/official-images/compare/master...docker-library-bot:joomla, and it *might* let you open the PR directly from there, but our experience with outside users being able to use those branches for PR opening directly has been really spotty.

**Comment by zero-24** [comment]
> When the bot detects changes, what it calculates the updated library/joomla to be will be visible at master...docker-library-bot:joomla, and it might let you open the PR directly from there, but our experience with outside users being able to use those branches for PR opening directly has been really spotty.

Ok just got that message:

image

I guess thats that spotty thing you mention right?

**Comment by J0WI** [comment]
> > When the bot detects changes, what it calculates the updated library/joomla to be will be visible at master...docker-library-bot:joomla, and it might let you open the PR directly from there, but our experience with outside users being able to use those branches for PR opening directly has been really spotty.
> 
> Ok just got that message:
> 
> image
> 
> I guess thats that spotty thing you mention right?

I have the same issue

**Pull request merged**